doc.js

```diff
@@ -225,41 +245,50 @@ define('doc', ['event'], function(event) {
 			},
 
 			'addClass' : function(clazz) {
+				var clazzList = clazz.split(' ');
 				this.each(function(el) {
-					if(!el.classList) {
-						if(!fallbackHasClass(el, clazz)) {
-							el.className += ' ' + clazz + ' ';
+					clazzList.forEach(function(clazz) {
+						if(!el.classList) {
+							if(!fallbackHasClass(el, clazz)) {
+								el.className += ' ' + clazz + ' ';
+							}
+						} else {
+							el.classList.add(clazz);
```

Comment:
`classList.add` seems to support receiving a list of classes to be added. What about using it instead?

Reply:
![image](https://user-images.githubusercontent.com/1116568/30566504-b13cb8a4-9ca2-11e7-8fbb-fc89fe22b79a.png)

Reply:
😢